environment/prod/deployment/prod/.env.prod

```diff
@@ -0,0 +1,29 @@
+COMPOSE_PROJECT_NAME=blumilk-website-prod
+
+BLUMILK_WEBSITE_HOST_NAME=blumilk.test.pl
+APP_URL="https://${BLUMILK_WEBSITE_HOST_NAME}"
+
+APP_NAME=blumilk-website
+APP_ENV=prod
+APP_DEBUG=false
+
+DB_CONNECTION=pgsql
+DB_HOST=blumilk-website-prod-database
+DB_PORT=5432
+
+BROADCAST_DRIVER=log
+CACHE_DRIVER=redis
+FILESYSTEM_DISK=local
+QUEUE_CONNECTION=redis
+SESSION_DRIVER=redis
+SESSION_LIFETIME=120
+
+REDIS_HOST=blumilk-website-prod-redis
+REDIS_PORT=6379
+
+MAIL_MAILER=smtp
+MAIL_HOST=smtp.gmail.com
+MAIL_PORT=465
+MAIL_ENCRYPTION=tls
+MAIL_FROM_ADDRESS=no-reply@blumilk.pl
+MAIL_FROM_NAME="${APP_NAME}"
```

Comment:
I think it should be adjusted also, because `blumilk-website` doesnt look good.

Reply:
Changed.